src/geff/networkx/io.py

```diff
@@ -107,14 +108,20 @@ def write(
 
     node_attrs = get_node_attrs(graph)
     if validate:
-        if position_attr not in node_attrs:
-            raise ValueError(f"Position attribute {position_attr} not found in graph")
-        for node, data in graph.nodes(data=True):
-            if position_attr not in data:
-                raise ValueError(f"Node {node} does not have position attribute {position_attr}")
+        if position_attr is not None:
+            if position_attr not in node_attrs:
+                raise ValueError(f"Position attribute {position_attr} not found in graph")
+            for node, data in graph.nodes(data=True):
+                if position_attr not in data:
+                    raise ValueError(
+                        f"Node {node} does not have position attribute {position_attr}"
```

Comment:
If we are allowing the position attribute to be totally missing, we should probably also allow it to be partially missing. Right?